app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: str) -> str:
```

Comment:
The `cafe` parameter should be of type `Cafe`, not `str`. Update the type annotation to `cafe: Cafe` to reflect that `cafe` is an instance of the `Cafe` class.